modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -156,84 +177,145 @@ class NDB_Form_Imaging_Browser extends NDB_Form
+                'processing:total_rejected'
             );
+            $SourceFile         = basename(
+                $FileObj->getParameter('processing:sourceFile')
+            );
+            $Tool = $FileObj->getParameter('Tool');
+            $SlicewiseRejected     = $this->_getRejected(
+                $FileObj,
+                'slicewise',
+                $sliceRej
+            );
+            $InterlaceRejected     = $this->_getRejected(
+                $FileObj,
+                'interlace',
+                $laceRej
+            );
+            $IntergradientRejected = $this->_getRejected(
+                $FileObj,
+                'intergradient',
+                $interRej
+            );
+            $Xstep       = number_format($FileObj->getParameter('xstep'), 2);
+            $Ystep       = number_format($FileObj->getParameter('ystep'), 2);
+            $Zstep       = number_format($FileObj->getParameter('zstep'), 2);
+            $Selected    = $this->_getSelected(
+                (int)$FileObj->getParameter('FileID')
+            );
+            $QCStatus    = $FileObj->getParameter('QCStatus');
+            $QCDate      = $FileObj->getParameter('QCLastChangeTime');
+            $Caveat      = $FileObj->getParameter('Caveat');
+            $SNR         = $FileObj->getParameter('SNR');
+            $SeriesUID   = $FileObj->getParameter('SeriesUID');
+            $XMLreport   = $FileObj->getParameter('processing:DTIPrepXmlReport');
+            $XMLprotocol = $FileObj->getParameter('ProtocolFile');
+            $NrrdFile    = $FileObj->getParameter('processing:nrrd_file');
+            $OtherTimepoints = $this->_getFilesAcrossTimepoints(
+                (int)$FileObj->getParameter('SessionID'),
+                (int)$FileObj->getParameter('AcquisitionProtocolID')
+            );
+
+            $file = array(
+                     'FileID'                => $FileID,
+                     'Filename'              => $Filename,
+                     'CheckPic'              => $CheckPic,
+                     'FullFilename'          => $FullFilename,
+                     'JivFilename'           => $JivFilename,
+                     'JivAddress'            => $JivAddress,
+                     'New'                   => $New,
+                     'Pipeline'              => $Pipeline,
+                     'OutputType'            => $OutputType,
+                     'AcquisitionProtocol'   => $AcquisitionProtocol,
+                     'CoordinateSpace'       => $CoordinateSpace,
+                     'Algorithm'             => $Algorithm,
+                     'AcquisitionDate'       => $AcquisitionDate,
+                     'ProcDate'              => $ProcDate,
+                     'FileInsertDate'        => $FileInsertDate,
+                     'SeriesDescription'     => $SeriesDescription,
+                     'SeriesNumber'          => $SeriesNumber,
+                     'EchoTime'              => $EchoTime,
```

Comment:
@kongtiaowang 

you forgot to add Repetition time, slice thickness, and time to the $file array